### ISSUE TITLE
running witch --check causes error

### ISSUE THREAD

**Issue by michal-autin** [FlorianKempenich/TOOCS-nodejs#1]
Hi.

I was trying your role, but I got stuck during 'ansible-playbook ... -C' run, looks like task: 'set_fact' is not able to carry playbook check:

 FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'stdout'\n\nThe error appears to have been in '/vagrant/ansible_roles/FlorianKempenich.nvm-node-npm/tasks/main.yml': line 23, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact:\n  ^ here\n"}

**Comment by HelloThisIsFlo** [comment]
Yes indeed there was a problem.

When running in `check_mode`, the following `shell` script wouldn't run:
```shell
if [ -s "{{ nvm_dir }}/nvm.sh" ]; then
    . {{ nvm_dir }}/nvm.sh && nvm --version
else
    echo 0
fi
```

Because the script wouldn't run, the `result` register would be empty, resulting in the error you described.

Since the script is merely to check the current (if existing) version of nvm, it has no side effect. I have updated the task list to for the execution of this script even in check mode.

I just pushed, it should work now ;)
Thanks for the issue.
